etc/ci/performance/README.md

```diff
@@ -0,0 +1,66 @@
+Servo Page Load Time Test
+==============
+
+# Prerequisites
+
+* Python3
+
+# Basic Usage
+
+## Prepare the test runner
+
+* Clone this repo
+* Download [tp5n.zip](http://people.mozilla.org/~jmaher/taloszips/zips/tp5n.zip), extract it to `page_load_test/`
+* Run `prepare_manifest.sh` to transform the tp5n manifest to our format
+* `python3 -m virtualenv venv; source venv/bin/activate; pip install treeherder-client`
```

Comment:
Instead of requiring a virtualenv, let's do something along the lines of:
```
Install the Python3 treeherder-client library.
For example, to install in a virtualenv, do: [snip]
```